tests/widgets_table.rs

```diff
@@ -399,26 +399,28 @@ fn widgets_table_columns_widths_can_use_mixed_constraints() {
         ]),
     );
 
-    // columns of large size (>100% total) hide the last column
-    test_case(
-        &[
-            Constraint::Percentage(60),
-            Constraint::Length(10),
-            Constraint::Percentage(60),
-        ],
-        Buffer::with_lines(vec![
-            "┌────────────────────────────┐",
-            "│Head1             Head2     │",
-            "│                            │",
-            "│Row11             Row12     │",
-            "│Row21             Row22     │",
-            "│Row31             Row32     │",
-            "│Row41             Row42     │",
-            "│                            │",
-            "│                            │",
-            "└────────────────────────────┘",
-        ]),
-    );
+    // This test is unstable and should not be in the test suite
```

Comment:
Was it unstable before this change? I don't think we've seen test failures, but perhaps it has been always cached and so never indeterminate? Can you cut an issue for this to investigate it at some point?

Reply:
It was unstable but rarely failed. But if you look through the recent PRs this test has changed a number of times by one column. I talk more about unstable solutions in #788

Reply:
Makes sense.